Dockerfile-py2

```diff
@@ -0,0 +1,16 @@
+FROM travisci/ci-garnet:packer-1496954857
+
+ENV PYTHON=2.7
+ENV NO_GIT_FETCH=1
+ENV PATH="/home/travis/miniconda/bin:${PATH}"
+ENV MINICONDA_FILENAME="Miniconda3-latest-Linux-x86_64.sh"
+ENV CONDA_REQS="conda=4.3.21 conda-env=2.6.0 conda-build=2.1.15"
+
+ADD . /tmp/bokeh
+RUN chown -R travis /tmp/bokeh
+
+USER travis
+
+RUN cd /tmp/bokeh && scripts/ci/install:test
+
+RUN rm -rf /tmp/bokeh
```

Comment:
OK .Maybe things could be moved but that's a nontrivial undertaking to coordinate, so probably not worth doing at this point. If it would not be too unusual for docker users  I'd prefer shorter and lowercase filenames like `docker-py2` and `docker-py3`

I'm also interested in centralizing conda req and miniconda filename so that that is not repeated in (at least) three places. Can docker files read this from other files? Or only env vars?